app/system_rollout.go

```diff
@@ -88,9 +88,10 @@ func systemRollout(c *cli.Context) error {
 	}
 
 	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, time.Second*30)
+	kubeFilteredInformerFactory := informers.NewFilteredSharedInformerFactory(kubeClient, time.Second*30, namespace, nil)
```

Comment:
Looks like we forgot to start kubeInformerFactory
```
go kubeInformerFactory.Start(doneCh)
```

Reply:
Also, looks like `NewFilteredSharedInformerFactory` is deprecated https://github.com/longhorn/longhorn-manager/blob/cf7b3e97346de9e677cf66081bf56cfcbb36d302/k8s/pkg/client/informers/externalversions/factory.go#L83-L89 Should we use [NewSharedInformerFactoryWithOptions](https://github.com/longhorn/longhorn-manager/blob/cf7b3e97346de9e677cf66081bf56cfcbb36d302/k8s/pkg/client/informers/externalversions/factory.go#L92-L93) instead

Reply:
Yes, updated. Thank you.